psi4/__init__.py

```diff
@@ -71,11 +77,19 @@
 # Make official plugins accessible in input
 from .driver import endorsed_plugins
 
+# Manage threads. Must be after endorsed plugins, honestly.
+core.set_num_threads(1, quiet=True)
+
```

Comment:
I'd still like `print_header()` to happen somewhere in psiapi mode. But admittedly, I can't find a place for it that wouldn't be annoying or unpythonic in some circumstance.